src/current/cockroachcloud/upgrade-cockroach-version.md

```diff
@@ -0,0 +1,54 @@
+Maintenance operations that are critical for cluster security or stability may be applied outside of the maintenance window, and upgrades that begin in a maintenance window may not always be completed by the end of the window.
+{{site.data.alerts.end}}
+
+Refer to [Cloud Upgrade Policy]({% link cockroachcloud/upgrade-policy.md %}).
+
+## Perform a major-version upgrade
+
+This section shows how to perform a major-version upgrade for a cluster in CockroachDB {{ site.data.products.cloud }}. Patch upgrades within a cluster's major version are applied automatically, and no action is required.
+
+1. Verify the cluster's current major version, and which versions it can be upgraded to:
+    1. Sign into [CockroachDB {{ site.data.products.cloud }}](https://cockroachlabs.cloud).
+    1. From the **Clusters** page, find the cluster by name. If the cluster is in a folder, click the name of the folder to view its descendants. The cluster's major version and patch are shown in the **Version** column.
+1. Check which upgrades are available to the cluster, if any.
+
+    Beginning with v24.1, major versions that end in `.1` and `.3` are required Regular releases, which can be upgraded to either the the next Innovation release or the next Regular release. For example, a cluster running v24.1 (a Regular release) can be upgraded directly either v24.2 (the next Innovation release) or v24.3 (the next Regular release). A cluster running v24.2 must be upgraded to v24.3.
+
+    To check whether major-version upgrades are available, click the three-dot **Action** menu. If upgrades are available, **Upgrade major version** will be enabled. Click it. In the dialog, if only one newer major version upgrade is available, it is selected automatically. If multiple upgrades are available, the latest Regular release is selected by default. If you intend to upgrade the cluster, keep this dialog open while completing the next steps. Otherwise, close the dialog.
+1. Before beginning a major-version upgrade, review its [Release notes]({% link releases/index.md %}), as well as the release notes for any skipped Innovation Releases. If any backward-incompatible changes or new features impact the cluster's workloads, you may need to make adjustments before beginning the upgrade.
+1. A CockroachDB {{ site.data.products.standard }} or {{ site.data.products.basic }} cluster remains fully available while it is upgraded. For a multi-node CockroachDB {{ site.data.products.advanced }} cluster, each cluster node is upgraded one at a time in a rolling fashion, and while a node is upgraded, the node's resources are temporarily unavailable to the cluster. A single-node CockroachDB {{ site.data.products.advanced }} cluster will be unavailable while the cluster is upgraded and restarted. If necessary, prepare for the upgrade by communicating ahead of time with your users, and plan to begin the upgrade during a time when the impact on the cluster's workload will be minimized.
```

Comment:
On a multi-node Advanced cluster, all data may remain accessible during upgrades and apps communicating with the DB may continue to work as normal, right? Perhaps with some additional latency?

I think we are giving a potentially false impression that you will see downtime on Adavnced. (Readers may not know how to interpret "while a node is upgraded, the node's resources are temporarily unavailable to the cluster. ")

Is it worth rewording all this?

Similar question re. the similar rollback mention.

Reply:
Done in both places